app/src/main/res/values/strings_account.xml

```diff
@@ -13,6 +13,9 @@
     <string name="account_error_user_not_found">User account not found.</string>
     <string name="account_error_unknown">Error logging in.</string>
     <string name="account_error_dialog_dismiss">OK</string>
+    <string name="account_error_not_connected">Please check your internet connection and try again.</string>
+    <string name="account_error_not_connected_title">No Internet Connection</string>
+    <string name="account_error_title">Authorization Failed</string>
```

Comment:
make this `Authentication Failed`

Reply:
authentication is the process of providing credentials to authenticate.

Authorization is a permission issue after you have already authenticated